MAUI/DataGrid/Column-types.md

```diff
@@ -1418,6 +1424,168 @@ The `DataGridNumericColumn` allows formatting the numeric data with culture-spec
+                       NavigationMode="Row">
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridMultiColumnComboBoxColumn AutoGenerateColumnsMode = "None"
+                                                      MappingName = "CustomerID"
+                                                      ItemsSource = "{Binding OrderDetails}"
+                                                      DisplayMember = "CustomerID"
+                                                      ValueMember = "CustomerID" 
+                                                      HeaderText="Customer ID">
+            <syncfusion:DataGridMultiColumnComboBoxColumn.Columns>
+                <syncfusion:DataGridTextColumn MappingName = "CustomerID" HeaderText = "Customer ID"/>
+                <syncfusion:DataGridNumericColumn MappingName = "ProductID" HeaderText = "Product ID"/>
+            </syncfusion:DataGridMultiColumnComboBoxColumn.Columns>
+        </syncfusion:DataGridMultiColumnComboBoxColumn>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+var column = new DataGridMultiColumnComboBoxColumn()
+{
+    MappingName = "CustomerID",
+    HeaderText = "Customer ID",
+    ValueMember = "CustomerID",
+    DisplayMember = "CustomerID",
+    ItemsSource = viewModel.OrderDetails,
+    AutoGenerateColumns = false,
+    Columns = new ColumnCollection()
+    {
+        new DataGridTextColumn(){MappingName = "CustomerID"},
+        new DataGridNumericColumn(){MappingName = "ProductID"}
+    }
+};
+this.dataGrid.Columns.Add(column);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+SfDataGrid triggers, (CurrentCellDropDownSelectionChanged)[] event, when the SelectedValue is changed. (CurrentCellDropDownSelectionChangedEventArgs)[] of `CurrentCellDropDownSelectionChanged` event provides the information about the changed cell value.
+
+`SelectedIndex` property returns the index of selected item.
+`SelectedItem` property returns the selected item from drop down list.
+
+<img alt="DataGridMultiColumnComboBox column .NET MAUI DataGrid" src="Images\column-types\maui-datagrid-MultiColumn-ComboBox-column.png" width="404" height="396"/>
+
+### Auto-complete support
+
+You can allow SfMultiColumnComboBox to complete the entered input value automatically by setting the (AllowAutoComplete)[] property to `true`. `AllowAutoComplete` property will autocomplete the input value by comparing it to each item of the underlying data source of `DataGridMultiColumnComboBoxColumn` and returns the matched value by DisplayMember.
+
+### Filtering
+
+You can allow SfMultiColumnComboBox to filter the drop-down list items dynamically based on the text typed on editor by setting (AllowIncrementalFiltering)[] property to `true`. DataGridMultiColumnComboBoxColumn allows you to filter the items based on case by setting (AllowCaseSensitiveFiltering)[] to `true`. This will help users to select from large number of items.
```

Comment:
You can enable the SfMultiColumnComboBox to dynamically filter the drop-down list items based on the text typed in the entry by setting the AllowIncrementalFiltering property to true. Additionally, DataGridMultiColumnComboBoxColumn allows filtering based on case sensitivity by setting the AllowCaseSensitiveFiltering property to true. These features help users quickly select items from a large list